emsdk.py

```diff
@@ -2623,6 +2624,8 @@ def construct_env(tools_to_activate, permanent):
   if to_unix_path(os.environ.get('EM_CONFIG', '')) != to_unix_path(em_config_path):
     env_vars_to_add += [('EM_CONFIG', em_config_path)]
   if emscripten_config_directory == emsdk_path():
+    # Remove this once emscriptensupport EM_CACHE in config file:
```

Comment:
I thought it would just support `CACHE`, not `EM_CACHE`? (wasn't the latter the env var name? maybe I'm misremembering)

Reply:
All config settings can be overridden via environment variables.  The pattern is that for a config settings called `FOO` the environment variable is `EM_FOO`.   So depending on which you are talking about they are both true.

Reply:
Fixed the comment.

Reply:
Thanks, right, I forgot about that env var stuff...